datalad/support/gitrepo.py

```diff
@@ -945,10 +946,11 @@ def push(self, remote=None, refspec=None, progress=None, all_=False,
                 #       with rm.repo.git.custom_environment(GIT_SSH="wrapper_script"):
                 with rm.repo.git.custom_environment(
                         GIT_SSH_COMMAND="ssh -S %s" % cnct.ctrl_path):
-                    rm.push(refspec=refspec, progress=progress, **kwargs)
+                    return rm.push(refspec=refspec, progress=progress,
+                                   **kwargs)
```

Comment:
with return here then it will quit before going through all remotes_to_push.  we do want to support multiple remotes to push here?